src/pages/settings/bank-accounts/common/hooks/useBankAccountColumns.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Link } from '@invoiceninja/forms';
+import { route } from 'common/helpers/route';
+import { useFormatMoney } from 'common/hooks/money/useFormatMoney';
+import { useCurrentCompany } from 'common/hooks/useCurrentCompany';
+import { BankAccount } from 'common/interfaces/bank-accounts';
+import { DataTableColumns } from 'components/DataTable';
+import { useTranslation } from 'react-i18next';
+
+export const useBankAccountColumns = () => {
+  const { t } = useTranslation();
+  const company = useCurrentCompany();
+  const formatMoney = useFormatMoney();
+
+  const columns: DataTableColumns<BankAccount> = [
+    {
+      id: 'bank_account_name',
+      label: 'Bank account name',
```

Comment:
Application will support multiple languages. You should never use raw strings for the labels or translations. Make use of `t` function that you get by `useTranslation` hook to pass the translation string.

If the string is missing please let me or David know, we will implement it on the API.

Reply:
> Application will support multiple languages. You should never use raw strings for the labels or translations. Make use of `t` function that you get by `useTranslation` hook to pass the translation string.
> 
> If the string is missing please let me or David know, we will implement it on the API.

Fixed